src/main/java/edu/rpi/legup/ui/proofeditorui/treeview/TreeView.java

```diff
@@ -596,89 +613,101 @@ public void calculateViewLocations(TreeNodeView nodeView, int depth) {
         dimension.width = Math.max(dimension.width, xLoc);
 
         TreeTransitionView parentTransView = nodeView.getParentView();
-        int yLoc = parentTransView == null ? (int) nodeView.getSpan() / 2 : parentTransView.getEndY();
+        int yLoc =
+                parentTransView == null ? (int) nodeView.getSpan() / 2 : parentTransView.getEndY();
         nodeView.setY(yLoc);
 
         ArrayList<TreeTransitionView> children = nodeView.getChildrenViews();
         switch (children.size()) {
             case 0:
                 break;
-            case 1: {
-                TreeTransitionView childView = children.get(0);
-
-                List<TreeNodeView> parentsViews = childView.getParentViews();
-                if (parentsViews.size() == 1) {
-                    childView.setEndY(yLoc);
+            case 1:
+                {
```

Comment:
Was messing around with CheckStyle myself and came across this. The autoformatter should be modified so that { is flush with the case statement. So like this:

```
case 1:
{
```

Reply:
It seems like once this is fixed, the Checkstyle will work properly